example/testapp/othermodule.py

```diff
@@ -39,8 +39,8 @@ def intro():
     
     return u'はじめまして'
 
-@rpcmethod(name='view.request')
-def request(**kwargs):
+@rpcmethod(name='view.getrequest')
+def getrequest(**kwargs):
```

Comment:
If you plan on changing this name, you need to update https://github.com/rpc4django/rpc4django/blob/f534a82/example/testapp/__init__.py#L4 as well.